### PR TITLE
fix getColumnTitle bug

### PR DESCRIPTION
**Pull request by geraldchen890806**
https://codesandbox.io/s/n7kwz8z8y0

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your pull request, thank you!

* [ ] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a pull request to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you pull request.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/commit/4b6d2ede2fbf167d45eca0afed4646b30805f462#commitcomment-31615118

**Comment by afc163** [comment]
补个用例

**Comment by geraldchen890806** [comment]
https://codesandbox.io/s/n7kwz8z8y0

File: components/table/Table.tsx [hunk 1]
```diff
@@ -882,8 +882,8 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
     }
     if (!(title instanceof Function) && typeof title !== 'string' && typeof title !== 'number') {
```
Comment (zy410419243) [inline components/table/Table.tsx]: `if (!React.isValidElement(title)) {`

**Review by orzyyyy — APPROVED** [review]
if (!React.isValidElement(title)) {

**Comment by geraldchen890806** [comment]
这个更新改出的bug。。。
优化鼠标悬停在可排序的表头上时 title 的显示。[#13312](https://github.com/ant-design/ant-design/issues/13312)

**Comment by afc163** [comment]
我来补用例吧。

**Comment by afc163** [comment]
will close #13542

**Pull request merged**